LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -81,6 +99,7 @@ public async Task<bool> UpdateReportedPropertiesAsync(TwinCollection reportedPro
                 }
 
                 this.logger.LogDebug("updating twin");
+                using var updateReportedPropertiesOperation = this.tracing.TrackIotHubDependency(UpdateReportedProperteisDependencyName, this.deviceIdTracingData);
```

Comment:
```suggestion
                using var updateReportedPropertiesOperation = this.tracing.TrackIotHubDependency(UpdateReportedPropertiesDependencyName, this.deviceIdTracingData);
```

Reply:
Done in 67ed957c5975e2697a55b52c0885f4941e52674f